src/components/Copyable.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+    <div class="copyable" :class="{ copied }" @click="copy">
+        <div class="background"></div>
+        <slot></slot>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import { Clipboard } from '@nimiq/utils';
+
+/**
+ * **Copyable**
+ *
+ * Copyable can be used to make a click on one or more elements copy content to the Clipboard with visual feedback.
+ * By default the children's contents are copied to the Clipboard. Alternatively, a specific text to be copied can be
+ * provided.
+ *
+ * Props:
+ *
+ * **text** {string} [optional] - A specific text to be copied to the clipboard
+ */
+@Component
+export default class Copyable extends Vue {
+    private static readonly DISPLAY_TIME = 2000;
+
+    @Prop(String)
+    public text?: string;
+
+    private copied: boolean = false;
+    private _copiedResetTimeout: number | null = null;
+
+    public copy() {
+        // Note that when iterating over childNodes, pseudo elements are excluded.
```

Comment:
While helpful in general I think this no longer belongs here.

Reply:
Thanks, removed it